pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -27,6 +27,9 @@ self: super: {
     dependencies = with super; [ vim-addon-manager ];
   };
 
+  # Mainly used as a dependency for fzf-vim. Wraps the fzf program as a vim
+  # plugin, since part of the fzf vim plugin is included in the main fzf
+  # program.
   fzfWrapper = buildVimPluginFrom2Nix {
```

Comment:
Do we need to expose it then to the user?

Reply:
Just having `fzf-vim` there would create less confusion what the right package is.

Reply:
I think it makes sense to keep it because

1. Backwards compatibility. No need to break the configuration of some users who have added `fzfWrapper`.
2. According to [upstream's readme](https://github.com/junegunn/fzf.vim):

> fzf in itself is not a Vim plugin, and the official repository only provides the basic wrapper function for Vim and it's up to the users to write their own Vim commands with it

So that sounds like `fzfWrapper` is intended to be used by end-users. To be honest I doubt that there are many users that do this, but it makes sense to support.

Reply:
In that case I'd keep `fzfWrapper` for now (at least) and do a proper deprecation phase.